gnd/src/main/java/com/google/android/gnd/ui/common/photoview/PhotoViewViewModel.java

```diff
@@ -0,0 +1,52 @@
+package com.google.android.gnd.ui.common.photoview;
+
+import android.net.Uri;
+import android.view.View;
+import androidx.databinding.ObservableInt;
+import androidx.lifecycle.LiveData;
+import androidx.lifecycle.MutableLiveData;
+import com.google.android.gnd.model.form.Field;
+import com.google.android.gnd.model.observation.Response;
+import com.google.android.gnd.system.StorageManager;
+import com.google.android.gnd.ui.common.AbstractViewModel;
+import javax.inject.Inject;
+
+public class PhotoViewViewModel extends AbstractViewModel {
```

Comment:
Naming: PhotoFieldViewModel, to make it clear that this is specific to photo fields in forms, not just any photo widget?